a3_finance/a3_finance/report/custom_general_ledger/custom_general_ledger.py

```
@@ -0,0 +1,724 @@
+			vouchers_to_ignore = (filters.get("voucher_no_not_in") or []) + [
+				x[0] for x in system_generated_cr_dr_journals
+			]
+			filters.update({"voucher_no_not_in": vouchers_to_ignore})
+
+	if filters.get("voucher_no_not_in"):
+		conditions.append("voucher_no not in %(voucher_no_not_in)s")
+
+	if filters.get("categorize_by") == "Categorize by Party" and not filters.get("party_type"):
+		conditions.append("party_type in ('Customer', 'Supplier')")
+
+	
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)